Gemfile.lock

```diff
@@ -669,7 +669,7 @@ GEM
     sidekiq-instrument (0.3.0)
       sidekiq (>= 4.2, < 6)
       statsd-instrument (~> 2.0, >= 2.0.4)
-    sidekiq-pro (3.7.0)
+    sidekiq-pro (3.7.1)
```

Comment:
@camerontesterman we're not ready to upgrade sidekiq pro yet. Please revert that before you merge. The vets_json_schema update is clearly part of this so no worries there.

Reply:
Ok, sounds good. I just pinned it to 3.7.0 to prevent that from occurring again until the platform team is ready to bump it.

Reply:
Thanks. We're dealing with the bigger upgrade of sidekiq separately.